PeerioTechnologies/peerio-desktop
# feat: pending files banner

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/9350/in-app-notice-regarding-downloading-pending-files

#### Testing instructions  
- make sure new account is not shown the banner
- make sure existing account are shown banners when have pending files
- make sure banner is not coming back with restart when dismissed

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
